packages/design-token-spec/lang/kdtl.pegjs

```diff
@@ -0,0 +1,216 @@
+{
+  function stringifyToken(token) {
+    switch (token.prefix) {
+      case '$scale':
+        return [token.prefix, token.target, token.name].join('/');
+      case '$semantic':
+        return [token.prefix, token.group, token.value].join('/');
+    }
+  }
+}
+
+KDT
```

Comment:
Done refactor on pegjs grammar.

After a benchmark, if the parser is too big, We can migrate to a lightweight parser generator like [reghex](https://github.com/kitten/reghex).
Our syntax should not contain too many features.